### DISCUSSION TITLE
Visual Studio Pyusb lib

### DISCUSSION THREAD

**Discussion by matt9687** [pyusb/pyusb#471] [Q&A] [answered]
Hi,
I am trying to use visual studio to develop my project but it seems that it can not achieve the library correctly.

In fact, when I am running the code it says: Import "usb.core" could not be resolved

there is someone that can help me?

Here the code I am using:

import time
import usb.core
import usb.util

VID = 0x04b4
PID = 0x00f3

dev = usb.core.find(idVendor=VID, idProduct=PID)
if not dev:
		print('Could not find FX3 :(')
		exit(1)
print('Yeeha! Found a FX3 devide')

// Send a control transfer
dev.ctrl_tranfer(reqType, bReq, wVal, wIndex, data)

// Connect to endpoint
exit(0)

#Image Visual studio
![image](https://user-images.githubusercontent.com/83287216/219327908-232b5555-112b-46ef-a71f-7e698ebc5e74.png)

**Comment by mcuee** [comment]
You have to put libusb-1.0.dll somewhere the Python installation can find.

**Reply by mcuee** [reply]
Read the FAQ as well.
https://github.com/pyusb/pyusb/blob/master/docs/faq.rst

**Reply by matt9687** [reply]
The libusb-1.0.dll was already installed in the Visual Studio directory.

![image](https://user-images.githubusercontent.com/83287216/220186157-dcdb123d-27ab-43db-987d-51750cf682e7.png)

**Comment by mcuee** [comment, marked as answer]
You may want to use Python installation downloaded from Python.org instead to see if that works.
